### PR TITLE
Added Assignment02 Files

### PR DESCRIPTION
**Pull request by deitesla**
Final submission of assignment two exercises.

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
needs review - should not have been merged.

File: students/lee_deitesfeld/lesson02/series.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    assert lucas(0) == 2
+    assert lucas(1) == 1
+    assert lucas(4) == 7
+    # test if sum_series matched fibonacci
+    assert sum_series(5) == fibonacci(5)
+
+    # test if sum_series matched lucas
+    assert sum_series(5, 2, 1) == lucas(5)
+
+    print("tests passed")
```
Comment (charlesrobison) [inline students/lee_deitesfeld/lesson02/series.py]: Nice assert statements!

File: students/lee_deitesfeld/lesson02/print_grid.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        print(pipe, space_by_four, pipe, space_by_four, pipe)
+        print(pipe, space_by_four, pipe, space_by_four, pipe)
+        print(pipe, space_by_four, pipe, space_by_four, pipe)
+        print(pipe, space_by_four, pipe, space_by_four, pipe)
+        print(plus, hyphen_by_four, plus, hyphen_by_four, plus)
+        print(pipe, space_by_four, pipe, space_by_four, pipe)
+        print(pipe, space_by_four, pipe, space_by_four, pipe)
+        print(pipe, space_by_four, pipe, space_by_four, pipe)
+        print(pipe, space_by_four, pipe, space_by_four, pipe)
+        print(plus, hyphen_by_four, plus, hyphen_by_four, plus)
```
Comment (charlesrobison) [inline students/lee_deitesfeld/lesson02/print_grid.py]: Not clear what you're doing here with a grid size of 9...?

File: students/lee_deitesfeld/lesson02/print_grid.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    else:
+        #prints one line with hyphens and plus signs
+        print(plus + hyphen_by_n + plus + hyphen_by_n + plus)
+        #prints multiple lines with pipes and spaces
+        for i in range(n):
+            print(pipe_line)
+        print(plus + hyphen_by_n + plus + hyphen_by_n + plus)
+        for i in range(n):
+            print(pipe_line)
+        print(plus + hyphen_by_n + plus + hyphen_by_n + plus)
```
Comment (charlesrobison) [inline students/lee_deitesfeld/lesson02/print_grid.py]: Creative solution

**Pull request merged**